tests/test_api.py

```diff
@@ -27,12 +27,80 @@ def test_scheduler_info(self):
         self.assertEqual(info['allowed_hosts'], ['*'])
         self.assertTrue(info['running'])
 
+    def test_pause_scheduler(self):
+        response = self.client.post(self.scheduler.api_prefix + '/pause')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_PAUSED)
+
+    def test_pause_scheduler_already_paused(self):
+        self.scheduler.pause()
+        self.assertEqual(self.scheduler.state, STATE_PAUSED)
+
+        response = self.client.post(self.scheduler.api_prefix + '/pause')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_PAUSED)
+
+    def test_resume_scheduler(self):
+        self.scheduler.pause()
+        self.assertEqual(self.scheduler.state, STATE_PAUSED)
+
+        response = self.client.post(self.scheduler.api_prefix + '/resume')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_RUNNING)
+
+    def test_resume_scheduler_already_running(self):
+        response = self.client.post(self.scheduler.api_prefix + '/resume')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_RUNNING)
+
+    def test_start_scheduler(self):
+        self.scheduler.shutdown()
+        self.assertEqual(self.scheduler.state, STATE_STOPPED)
+
+        response = self.client.post(self.scheduler.api_prefix + '/start')
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(self.scheduler.state, STATE_RUNNING)
+
+    def test_start_scheduler_already_running(self):
+        response = self.client.post(self.scheduler.api_prefix + '/start')
+        self.assertEqual(response.status_code, 400)
+
+    def test_shutdown_scheduler(self):
+        self.client.post(self.scheduler.api_prefix + '/start')
```

Comment:
The scheduler is already started in the `setUp` method, no need to call `/start` here